syntax_and_semantics/cross-compilation.md

```diff
@@ -2,7 +2,20 @@
 
 Crystal supports a basic form of [cross compilation](http://en.wikipedia.org/wiki/Cross_compiler).
 
-In order to achieve this, the compiler executable provides two flags:
+## Preparation
+
+Cross-compilation will depend on `libcrystal` being present on the target system, so we need to build it by:
```

Comment:
Direct address ("we") should be avoided. I know, it's used all over the place but let's try to remove it.
Instead of `, so we need to build it by:` you could write `. It can ne built following these steps:`

Reply:
I mixed style because it was mixed in this document, will adjust as you suggested though!